doc/theory/glossary.tex

```diff
@@ -287,6 +287,22 @@
 	
 	The name is inspired from the case when the persistence diagram comes from persistent homology.
 	
+	\subsection*{Heat kernel:} \label{heat kernel}
+	
+	Given a set $\xi$, a kernel $k:\xi \times \xi \rightarrow \mathbb{R}$ is a kernel, if there exists a Hilbert space $H$ with inner product $\langle \cdot, \cdot \rangle$ and a function $\phi: \xi \rightarrow H$, such that $k(x,y) = \langle \phi(x), \phi(y) \rangle$.
+	
+	In contrast to vectorisation methods, kernel methods do not give an explicit feature map nor a vector representation. Nevertheless,such methods remain popular for tasks like classification.
+	
+	Let $D, D'$ be two \hyperref[persistence diagram]{persistence diagrams}. Then, the persistence scale-space kernel, often called the \textit{heat kernel}, is defined by
+	\begin{equation*}
+	k_\sigma(D,\, D') = \frac{1}{8\pi\sigma}\sum_{p\in D,\, q \in D'}
+	\exp\left(\frac{-\Vert p-q\Vert^2}{8\sigma}\right) -
+	\exp\left(\frac{-\Vert p-\bar{q}\Vert^2}{8\sigma}\right),
+	\end{equation*}
+	where $\bar{q} = (d,b)$ for a point $q=(b,d)$.
```

Comment:
But this means $\bar{q}=q$ so why can't we just use $q$?

Reply:
I'm sorry, I can't read. Please ignore.

Reply:
I wanted the 'bar' to denote the reflection across the diagonal, so that $\bar{q}=q$ is true for $q$ on the diagonal.
How would you suggest to phrase it?

Reply:
As I said, I had missed the swapping between d and b so please ignore this thread.